app/views/slack/send_notification.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= content_for :title, "Slack - Send Notification" %>
+
+<% if @notification_success %>
```

Comment:
Wondering if we need this page or if we should redirect to the reporting_period show page with the notice message informing that the message was sent to the user. Otherwise if an admin needs to send it to many people this will become a bit tedious.

Reply:
If we keep this page, let's add some link to go back to the reporting period.

Reply:
I agree it makes sense to return to the same view after sending the Slack message, otherwise it might become tedious, yeah!

I updated the controller to redirect to the reporting period page, with a notice in case of success and an alert in case of error, let me know if this works for you!

Reply:
(and due to that, I removed this view, since it's no longer necessary)